src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -1092,50 +834,6 @@ public FormValidation doCheckLabelString(
       return FormValidation.ok();
     }
 
-    public ListBoxModel doFillWindowsPasswordCredentialsIdItems(
```

Comment:
Sanity checking here, were these doFills intentionally removed or were they accidently swept up in the re-factor?

Reply:
oic, they were moved down to WindowsConfig. Disregard.